Solutions/Corvus.Json.SchemaGenerator/Program.cs

```diff
@@ -17,62 +17,82 @@ class Program
     {
         static Task<int> Main(string[] args)
         {
+            var rootNamespace = new Option<string>(
+                                "--rootNamespace",
+                                description: "The default root namespace for generated types");
+            var rootPath = new Option<string>(
+                                "--rootPath",
+                                description: "The path in the document for the root type.");
+            var useSchema = new Option<SchemaVariant>(
+                                "--useSchema",
+                                getDefaultValue: () => SchemaVariant.Draft201909,
+                                description: "The schema variant to use.");
+            var outputMapFile = new Option<string>(
+                                "--outputMapFile",
+                                description: "The name to use for a map file which includes details of the files that were written.");
+            var outputPath = new Option<string>(
+                                "--outputPath",
+                                description: "The output directory. It defaults to the same folder as the schema file.");
+            var outputRootTypeName = new Option<string?>(
+                                "--outputRootTypeName",
+                                getDefaultValue: () => null,
+                                description: "The Dotnet TypeName for the root type.");
+            var rebaseToRootPath = new Option<bool>(
+                                "--rebaseToRootPath",
+                                "If a --rootPath is specified, rebase the document as if it was rooted on the specified element.");
+
+            var schemaFile = new Argument<string>(
+                    "schemaFile",
```

Comment:
I don't know if you care about such things, but the indentation is inconsistent in this file. All the `Option<T>`s seem to be indented 20 spaces from their parent, but this is 8 spaces in.

(Bearing in mind that some of the `description` arguments are on the long side, I'd be inclined to indent them all by just 4, if you did change this.)